### PR TITLE
[ Onboarding - Nuova pagina profilo] Added new profile page, taking d…

### PR DESCRIPTION
**Pull request by RiccardoMolinari95**
Onboarding - Nuova pagina profilo

## Short description
Added new profile page, taking data from a separate stream. The new profile page displays:

- first and last name
- tax code
- email
- date of birth
The feature can be activated via the switch in Settings -> Preferences -> Enable new profile page
Enabling the feature will display a button with the profile icon in the navigator tab

## How to test
Enable "New Profile page"

### CODE REVIEW

**Review by hevelius — COMMENTED** [review]
What do you think to see the "new profile" a new feature in io-app? As you cans ee the codebase is organized by features. In the ts/features folder there are the mainly features of IO. For ex. the new profile, code, logic etc... could be in the ts/features/newProfile/ and protected behind a local feature flag

File: ts/features/newProfile/store/reducers/newProfile.ts
Comment (hevelius) [inline ts/features/newProfile/store/reducers/newProfile.ts:1]: What do you think to add a unit test for this reducer?

**Review by hevelius — CHANGES_REQUESTED** [review]

File: ts/features/newProfile/store/reducers/newProfile.ts
Comment (RiccardoMolinari95) [reply ts/features/newProfile/store/reducers/newProfile.ts:1]: Yes, sounds good

**Review by hevelius — APPROVED** [review]

**Comment by RiccardoMolinari95** [comment]
[ Onboarding - Nuova pagina profilo]

**Pull request merged**